functorch/csrc/BatchedTensorImpl.cpp

```diff
@@ -175,9 +170,12 @@ bool inplaceIsVmapCompatible(const Tensor& self, const Tensor& other) {
     // self is not batched but other is batched
     return false;
   }
-  auto self_levels = createVmapLevelsBitset(self_batched->bdims());
-  auto other_levels = createVmapLevelsBitset(other_batched->bdims());
-  return self_levels == (self_levels | other_levels);
+  // TODO(vfdev): As BatchedTensorImpl is refactored and has only one dim and one level.
+  // Below code may be simplified.
+  // auto self_levels = createVmapLevelsBitset(self_batched->level());
+  // auto other_levels = createVmapLevelsBitset(other_batched->level());
+  // return self_levels == (self_levels | other_levels);
+  return self_batched->level() == other_batched->level();
```

Comment:
I don't think this function is used anywhere actually. Could we delete it? (as well as the declaration in the header)